benchmarks/test_envs_benchmark.py

```diff
@@ -36,21 +40,92 @@ def make_parallel_env():
     return ((env,), {})
 
 
+def make_nested_td():
```

Comment:
If you want you can use some parts of test_nested in https://github.com/pytorch/rl/pull/1339/files for a more parametric creation of nested td with options to nest only some pats of it

Reply:
Seems too complex for a benchmark: the tensors are bigger and with random values, here we have just 0s and 1s and we just focus on the tensordict operations on keys etc

Reply:
oh yes i was just saying that maybe we want `nested_done`, `nested_reward`, `nested_action` as arguments instead of a general `nested`